targets/teensy40.json

```diff
@@ -0,0 +1,25 @@
+{
+  "inherits": ["cortex-m"],
+  "llvm-target": "armv7em-none-eabi",
+  "cpu": "cortex-m7",
+  "build-tags": ["teensy40", "teensy", "mimxrt1062", "nxp"],
+  "automatic-stack-size": false,
+  "default-stack-size": 131072,
+  "cflags": [
+    "--target=armv7em-none-eabi",
+    "-Qunused-arguments",
+    "-mfloat-abi=hard",
+    "-mfpu=fpv5-d16"
+  ],
+  "ldflags": [
+    "--emit-relocs",
+    "--gc-sections"
+  ],
```

Comment:
These flags are already defined in cortex-m.json so don't need to be repeated here.

Reply:
Thanks, wasn't sure how the inheritance worked among these JSON files - if it intelligently merges JSON arrays so that unique elements are preserved among both parent and child, or if it would override the parent if the child provided its own definition..

Anyhow, I think your comment implies that it *Does The Right Thing*. Will remove.